frontend/src/App.js

```diff
@@ -1,24 +1,47 @@
 import React from 'react';
-import { BrowserRouter, Route, Routes, Outlet, Navigate } from 'react-router-dom';
+import {
+  BrowserRouter,
+  Route,
+  Routes,
+  Outlet,
+  Navigate,
+} from 'react-router-dom';
 import { useSelector } from 'react-redux';
 import HomePage from './pages/Home';
 import NotFound from './pages/NotFound';
 import Navigation from './components/Navigation';
 import LoginPage from './pages/LoginPage';
+import SignupPage from './pages/SignupPage';
 import { UserContext } from './auth/userContext';
 
 function App() {
+  const { isAuthenticated } = useSelector((state) => state.auth);
 
   return (
     <UserContext>
       <BrowserRouter>
         <Routes>
           <Route element={<AnonymousRoute />}>
             {/* These routes only available to those NOT logged in */}
-            <Route exact path='/login' element={<LoginPage />}/>
-            <Route exact path='/signup' element={<h1>Sign up here</h1>}/>
+            <Route
+              exact
+              path='/login'
+              element={
```

Comment:
do we need to still wrap this in the anonymous route on line 24 or does this work without it